test/common/protobuf/utility_test.cc

```diff
@@ -23,4 +23,21 @@ TEST(UtilityTest, LoadProtoFromFile) {
   EXPECT_TRUE(TestUtility::protoEqual(bootstrap, proto_from_file));
 }
 
+TEST(UtilityTest, LoadTextProtoFromFile) {
+  envoy::api::v2::Bootstrap bootstrap;
+  bootstrap.mutable_cluster_manager()
+      ->mutable_upstream_bind_config()
+      ->mutable_source_address()
+      ->set_address("1.1.1.1");
+
+  std::string bootstrap_text;
+  ASSERT_TRUE(Protobuf::TextFormat::PrintToString(bootstrap, &bootstrap_text));
+  const std::string filename =
+      TestEnvironment::writeStringToFileForTest("proto.pb_text", bootstrap_text);
+
+  envoy::api::v2::Bootstrap proto_from_file;
+  MessageUtil::loadFromFile(filename, proto_from_file);
+  EXPECT_TRUE(TestUtility::protoEqual(bootstrap, proto_from_file));
```

Comment:
nit: please also test the exception case with EXPECT_THROW_WITH_MESSAGE by passing a bogus .pb_text file.